superset-frontend/src/explore/controlUtils/getFormDataWithDashboardContext.ts

```diff
@@ -194,14 +213,18 @@ export const getFormDataWithDashboardContext = (
     .reduce(
       (acc, key) => ({
         ...acc,
-        [key]: removeAdhocFilterDuplicates([
-          ...ensureIsArray(exploreFormData[key]),
-          ...ensureIsArray(filterBoxData[key]),
-          ...ensureIsArray(nativeFiltersData[key]),
-        ]),
+        [key]: applyTimeRangeFilters(
```

Comment:
I'm wondering if `mergeNativeFiltersToFormData` function (in the same file) isn't a better place for this transformation

Reply:
That was my initial thinking but I also want to cover filter box cases. `mergeNativeFiltersToFormData` does not give me easy access to the filters list, as it returns an object that may contain different keys such as `adhoc_filters` and `adhoc_filters_b`

Reply:
Got it 👍